index.html

```diff
@@ -519,6 +519,79 @@ <h3>LinkedDataSignature2016</h3>
           </pre>
       </section>
 
+
+      <section id="MerkleProof2019" about="https://w3id.org/security#MerkleProof2019">
+
+        <h3>MerkleProof2019</h3>
+          <p>
+A Linked Data signature is used for digital signatures on RDF Datasets.
+The default canonicalization mechanism is specified in the RDF Dataset
+Normalization specification, which effectively deterministically names all
+unnamed nodes. The default signature mechanism
+uses a SHA-256 digest and ECDSA to perform the digital signature. More information at
+              <a href="https://w3c-ccg.github.io/lds-merkle-proof-2019/">https://w3c-ccg.github.io/lds-merkle-proof-2019/</a>
+          </p>
+          <dl>
+            <dt>Status</dt>
+            <dd property="vs:term_status">stable</dd>
+            <dt>Parent Class</dt>
+            <dd>LinkedDataSignature</dd>
+            <dt>Expected properties</dt>
+            <dd>proofValue</dd>
+            <dt>Signature Properties</dt>
+            <dd>
+              <dl>
+                <dt>Default Canonicalization Algorithm</dt>
+                <dd>
+<a rel="canonicalizationAlgorithm"
+  href="https://w3id.org/security#GCA2015">https://w3id.org/security#GCA2015</a>
+                </dd>
+                <dt>Default Signature Algorithm</dt>
+                <dd>
+<a rel="signatureAlgorithm"
+  href="http://www.w3.org/2000/09/xmldsig#rsa-sha256">
+http://www.w3.org/2000/09/xmldsig#rsa-sha256</a>
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+          <p>
+The example below shows how a basic JSON-LD signature is expressed in a
+JSON-LD snippet. Note that the proof property is directly embedded in the
+object. The signature algorithm specifies how the signature can be generated
+and verified.
+          </p>
+          <pre class="example prettyprint language-jsonld">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.blockcerts.org/schema/3.0-alpha/context.json",
```

Comment:
You may want to move this to a more permanent domain than just Blockcerts. I suggest https://w3id.org/blockcerts/v1 and redirecting that to https://www.blockcerts.org/schema/3.0-alpha/context.json.

Reply:
example: https://github.com/perma-id/w3id.org/pull/1891

Reply:
yeah this is still WIP and not really available to the outside world. I used this document as an example (verifiable) v3 blockcerts because this is what I have access to at this moment. 

But point taken this is something that we need to do on the roadmap to deliver blockcerts v3. If you feel the example should be more sturdy I will update.

Reply:
I would recommend setting up w3id.org sooner rather than later, it will help if things needs to move and you don't want to break your proofs.

Reply:
+1, please take the 10 minutes it takes to do the PR for w3id.org -- it'll save you hours, to days, to weeks of pain in the future.

Reply:
@lemoustachiste wrote:
> yeah this is still WIP and not really available to the outside world.

@dsemenovsky wrote:
> Please, keep me in the loop as we are using this method for production purposes.

Haha -- so, which is it? Is this not available to the outside world *and* it's being used in production?

Reply:
I guess the confusion is caused by the fact we represent two different organizations, one uses it already and one doesn’t.

Reply:
Ah, ok, I get it now. Erm, so since someone has shipped this in production, how terrible would it be if the URL for the context changed and the `verificationMethod` value changed? I'll assert that it's technically possible to do this and not affect your production deployment (you'd just update your library with a new code path for the corrected specification and retail the old code path when you see the old `@context` value).

In the future, please don't ship things into production w/o getting proper review on your spec. It puts a burden on the community to start solving issues in suboptimal ways (and takes volunteer effort away from other things for the fire drill).

Reply:
For us changes in `verificationMethod` will not affect anything.

In terms of `@context` we are using `credentials/v1` in tandem with custom context defined, which I believe is the whole purpose of having an array of contexts there, so that you can have flexibility and scalability of the credentials form.

It is more related to verifying JSON-LD structure than the cryptographic verification.

Reply:
Yes, good.

Please change:

```
https://www.blockcerts.org/schema/3.0-alpha/context.json
```

to 

```
https://w3id.org/blockcerts/v3-alpha
```

and then setup the appropriate redirect at https://w3id.org/

Reply:
here is an example of how to setup a redirect with w3id.org: https://github.com/perma-id/w3id.org/pull/1891

Reply:
I will tend to that

Reply:
https://w3id.org/blockcerts/v3.0-alpha is now a thing, I am in the process of updating the example

Reply:
+1, wonderful!